src/components/Upload.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import styles from './Upload.scss';
+
+const renderDropzone = field => (
+  <div>
+    <Dropzone
+      accept="image/*"
+      maxSize="3000"
+      className={styles.overide}
+      name={field.name}
+      onDrop={fileToUpload => field.input.onChange(fileToUpload)}
```

Comment:
My questions was about the form field value. If I use this component in a form, what kind of value do a receive from this field in my `onSubmit` handler?

But we already discussed this. Currently it starts with an empty string, and then turns into a Javascript File object. This need to change to a dataUrl.